dev/vite.config.ts

```diff
@@ -10,5 +9,4 @@ export default mergeConfig(viteConfig, {
         // Dev/Storybook environment.
         "process.env.STORYBOOK": "true",
     },
-    plugins: [react({jsxRuntime: "automatic"})],
```

Comment:
see the `vite.config.ts` at the root of the repo - we include the react plugin there now.